### DISCUSSION TITLE
Height container query

### DISCUSSION THREAD

**Discussion by tcolinpa** [tailwindlabs/tailwindcss#11707] [Ideas]
I know there's the official tailwind plugin for container queries, however it only supports container queries related to the width, not height (afaik).

There's some discussions #10076 and one [PR got closed](https://github.com/tailwindlabs/tailwindcss-container-queries/pull/7).

One common case is styling anything related to time (think of a weekly grid calendar) or anything that you don't have full control over the element's height.

Is this something worth developing into the plugin or in this case it'd be better to just create a plugin directly in my tailwind config's project file?

**Comment by justinbmeyer** [comment]
I'm building a reporting tool that tries to fit everything in a 1900 x 1080 element.  This would be very useful.
